debian/intelmq.install

```diff
@@ -1,2 +1,3 @@
-CHANGELOG.md LICENSE COPYRIGHT MANIFEST.in REQUIREMENTS docs/*md usr/share/doc/intelmq/
```

Comment:
On Wed, Jul 20, 2016 at 10:01:01AM -0700, Sebastian wrote:
> The LICENSE file is removed here, but not added (in this bunch of
> commits). Is it still shipped?

Good eye!  It is being left out on purpose because it should also be
included in debian/copyright.

Unfortunately, it isn't yet.  I'm going to fix that right away.